### PR TITLE
Issue/2821/text area input docs

### PR DESCRIPTION
**Pull request by IanKBovard**
#### What does this PR do?
Adds to textArea and textInput docs.

#### Where should the reviewer start?
textArea/docs.js and textInput/docs.js

#### What testing has been done on this PR?
None

#### What are the relevant issues?
#2821 

#### Do the grommet docs need to be updated?
auto

#### Should this PR be mentioned in the release notes?
no

#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Comment by ShimiSun** [comment]
Yes, I agree with @oorestisime. @IanKBovard will you be willing to give it a try? It will help us reuse your definitions and be consistent on our docs.

**Comment by IanKBovard** [comment]
i gotcha. Would it be preferred to include the currently non functioning global.control.border.color in this, like the example above?

```
borderStyle: {
'global.control.border.width': {
    description: 'The border width.',
    type: 'string',
    defaultValue: '1px',
  },
  'global.control.border.radius': {
    description: 'The border radius.',
    type: 'string',
    defaultValue: '4px',
  },
  'global.control.border.color': {
    description: 'The border color.',
    type: 'string',
    defaultValue: 'border',
  }, 
}
```

Or would it be preferred to remove that  and add global.colors.border, like so?

```
borderStyle: {
'global.control.border.width': {
    description: 'The border width.',
    type: 'string',
    defaultValue: '1px',
  },
  'global.control.border.radius': {
    description: 'The border radius.',
    type: 'string',
    defaultValue: '4px',
  },
// change is here
  'global.colors.border': {
    description: 'The border color.',
    type: 'string',
    defaultValue: {
      dark: 'rgba(255, 255, 255, 0.33)',
      light: 'rgba(0, 0, 0, 0.33)',
    },
  }, 
}
```

**Comment by ShimiSun** [comment]
good question @IanKBovard. I like the second proposal as it is more direct and doesn't require the user to browse what `border` means and how it is translated into colors. but I'll ask for @ericsoderberghp opinion here as well.

**Comment by ericsoderberghp** [comment]
I think we should call the function `controlBorderStyle`, as that seems a bit more accurate, since Box borders don't use it.

Can we add `global.control.border.color = 'border'` to base.js? That would allow us to keep it focused on the control context, and permit changing the control border color independent of the Box border color. This is along the lines of #2806.

**Comment by IanKBovard** [comment]
I gotcha. there is a `global.control.border.color = 'border'` currently in base.js. Changing this property has no effect on the border, however. I posted this issue here #2900

**Comment by ShimiSun** [comment]
Hi @IanKBovard. issue #2900 is solved, please go ahead with the documentation of 
```
  'global.control.border.color': {
    description: 'The border color.',
    type: 'string',
    defaultValue: 'border',
  }, 
```

**Comment by IanKBovard** [comment]
I gotcha. Changes have been made.

**Comment by ShimiSun** [comment]
Need to resolve conflicts, please :)

File: src/js/components/TextArea/doc.js [hunk 1]
```diff
@@ -49,17 +49,23 @@ export const themeDoc = {
...
+    defaultValue: '4px',
+  },
+  'global.control.border.color': {
+    description: 'The border color.',
+    type: 'string',
+    defaultValue: 'border',
   },
   'global.colors.border': {
-    description: 'The color of the borders',
+    description: 'The color of the border',
```
Comment (ShimiSun) [inline src/js/components/TextArea/doc.js]: Can you please add a dot at the end of the line, it renders better on the README.md file.

**Comment by IanKBovard** [comment]
new pr opened using current master https://github.com/grommet/grommet/pull/2933

**Pull request closed without merging**